decidim-proposals/db/migrate/20170411173215_add_official_image_proposal_to_proposals.rb

```diff
@@ -0,0 +1,5 @@
+class AddOfficialImageProposalToProposals < ActiveRecord::Migration[5.0]
```

Comment:
Missing rename on Class name and filename: it should be: 
* file 20170411173215_add_image_to_proposals.rb 
* class AddImageToProposals